content/docs/glossary.js

```diff
@@ -1,89 +0,0 @@
-export default {
-  name: 'Glossary',
-  desc:
-    'This document is aimed to provide definitions for relevant DVC ' +
-    'concepts and terminologies that are frequently used.',
-  contents: [
-    {
-      name: 'Workspace',
-      match: ['workspace'],
-      desc: `
-Directory containing all your project files, for example raw datasets, source
-code, ML models, etc. It will conatain your DVC project.
-      `
-    },
-    {
-      name: 'DVC Project',
-      match: [
-        'DVC project',
-        'DVC projects',
-        'project',
-        'projects',
-        'DVC repository',
-        'DVC repositories'
-      ],
-      desc: `
-Initialized by running \`dvc init\` in the **workspace** (typically in a Git
-repository). It will contain the
-[\`.dvc/\` directory](/doc/user-guide/dvc-files-and-directories) and
-[DVC-files](/doc/user-guide/dvc-file-format) created with commands such as
-\`dvc add\` or \`dvc run\`.
-      `
-    },
```

Comment:
I think we missed migrating this one.

![image](https://user-images.githubusercontent.com/1477535/82844001-89168100-9ea4-11ea-890b-811236d33301.png)

![image](https://user-images.githubusercontent.com/1477535/82843985-8025af80-9ea4-11ea-8489-450895f0bd3c.png)

Reply:
Whoops! Thanks for the catch!

Reply:
Fixing it in #1358 🙂

Reply:
Great, thanks!